Dockerfile

```
@@ -1,20 +1,17 @@
-ENV REMOTE_SOURCES_DIR=${REMOTE_SOURCES_DIR:-.}
-ENV APP_DIR=.
 ENV CACHE_DEPS="true"
-WORKDIR /opt/apt-root/src
+WORKDIR /opt/app-root/src
 
 
-COPY ${APP_DIR}/api ./api
-COPY ${APP_DIR}/go.mod ${APP_DIR}/go.sum ./
+COPY ./api ./api
+COPY ./go.mod ./go.sum ./
 RUN if [ -n $CACHE_DEPS ]; then go mod download ; fi
```

Hadolint (2.14.0)

[error] -n doesn't work with unquoted arguments. Quote or use [[ ]].

(SC2070)